src/test/java/com/ifmo/epampractice/security/SecureStringTest.java

```diff
@@ -0,0 +1,105 @@
+package com.ifmo.epampractice.security;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.charset.StandardCharsets;
+
+public class SecureStringTest {
+
+    @Test
+    public void stringValueTest() {
+        char[] chars = new char[] { 'n', 'i', 'c', 'e' };
+        SecureString str = new SecureString(chars);
+
+        Assert.assertEquals(chars.length, str.getLength());
+
+        for (int i = 0; i < str.getLength(); i++) {
+            Assert.assertEquals(chars[i], str.charAt(i));
+        }
+
+        str.dispose();
+    }
+
+    @Test
+    public void stringDisposeTest() {
+        char[] chars = new char[] { 'n', 'i', 'c', 'e' };
+        SecureString str = new SecureString(chars);
+        str.dispose();
+
+        Assert.assertEquals(true, str.isDisposed());
```

Comment:
You are asserting that true equals boolean. How we can change it? Do we write if( a == true){}?

Reply:
Fixed in the following commit